### ISSUE TITLE
Crash when I start scanning  NoClassDefFoundError: android/telephony/CellIdentityCdma

### ISSUE THREAD

**Issue by agilob** [wish7code/openbmap#140]
```
W/dalvikvm(10238): threadid=1: thread exiting with uncaught exception (group=0x40fa28a8)
E/AndroidRuntime(10238): FATAL EXCEPTION: main
E/AndroidRuntime(10238): java.lang.NoClassDefFoundError: android/telephony/CellIdentityCdma
E/AndroidRuntime(10238):        at java.lang.Class.getDeclaredMethods(Native Method)
E/AndroidRuntime(10238):        at java.lang.Class.getDeclaredMethods(Class.java:703)
E/AndroidRuntime(10238):        at a.a.a.o.a(Unknown Source)
E/AndroidRuntime(10238):        at a.a.a.c.a(Unknown Source)
E/AndroidRuntime(10238):        at a.a.a.c.a(Unknown Source)
E/AndroidRuntime(10238):        at org.openbmap.services.wireless.WirelessLoggerService.a(Unknown Source)
E/AndroidRuntime(10238):        at org.openbmap.services.a.onCreate(Unknown Source)
E/AndroidRuntime(10238):        at org.openbmap.services.wireless.WirelessLoggerService.onCreate(Unknown Source)
E/AndroidRuntime(10238):        at android.app.ActivityThread.handleCreateService(ActivityThread.java:2537)
E/AndroidRuntime(10238):        at android.app.ActivityThread.access$1600(ActivityThread.java:151)
E/AndroidRuntime(10238):        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1383)
E/AndroidRuntime(10238):        at android.os.Handler.dispatchMessage(Handler.java:99)
E/AndroidRuntime(10238):        at android.os.Looper.loop(Looper.java:153)
E/AndroidRuntime(10238):        at android.app.ActivityThread.main(ActivityThread.java:5071)
E/AndroidRuntime(10238):        at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime(10238):        at java.lang.reflect.Method.invoke(Method.java:511)
E/AndroidRuntime(10238):        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:790)
E/AndroidRuntime(10238):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:557)
E/AndroidRuntime(10238):        at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime(10238): Caused by: java.lang.ClassNotFoundException: android.telephony.CellIdentityCdma
E/AndroidRuntime(10238):        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:61)
E/AndroidRuntime(10238):        at java.lang.ClassLoader.loadClass(ClassLoader.java:501)
E/AndroidRuntime(10238):        at java.lang.ClassLoader.loadClass(ClassLoader.java:461)
E/AndroidRuntime(10238):        ... 19 more
V/Provider/Setting(  582): from settings cache , name = dropbox:data_app_crash value = disabled
V/Provider/Setting(  582): from settings cac
````

It happens when I start new scanning session or open old one. Previous fdroid version worked fine.

**Comment by wish7code** [comment]
Hey agilob,

what Android version / ROM are you using?

Cheers
Toby

**Comment by agilob** [comment]
custom rom from LG android 4.1

**Comment by wish7code** [comment]
Seems to be a EventBus bug: https://github.com/greenrobot/EventBus/issues/149

I just pushed a new commit fixing the issue. In the emulator it now seems to runs fine on 4.1

**Comment by wish7code** [comment]
In case you need binaries I also updated https://6597aa1f72f81233f964cec56f26d45bc9ed86d3.googledrive.com/host/0B97gHr4MqjHpM2h0QVR5SWJOcGs/radiobeacon.apk

_
(Binaries disclaimer: as this is signed with my dev key, you'll have to uninstall F-droid version before being able to install)_

**Comment by agilob** [comment]
I was just about to fix it myself, I'll test it today

**Comment by agilob** [comment]
It works, please make a release to fdroid :)

**Comment by agilob** [comment]
Please make an update in FDroid.

**Comment by agilob** [comment]
I tested it once again and noticed that the .apk you provided doesn't start scanning at all and GPS fix is never requested from device, wardriving session is also never created.

**Comment by wish7code** [comment]
Argh, sorry forgot to update this issue. Indeed scanning didn't work in commit above. It was fixed until 28046e9302..

I'll push an updated binary. F-Droid update will still take a couple of days as I we still have a map display issue in the latest binary.

**Comment by agilob** [comment]
Please update fdroid ;/
